integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -0,0 +1,375 @@
+/**
+* Copyright 2018 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+require 'json'
+control 'explain' do
```

Comment:
Can you show me what is invoking these explain tests?

Reply:
All the controls are invoked by kitchen in `kitchen.yml`. Since there is only one suite and controls are not specified, all the controls are invoked.

Reply:
This makes the code cleaner.